pkgs/development/python-modules/miasm/default.nix

```
@@ -0,0 +1,51 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, future
+, gcc
+, llvmlite
+, parameterized
+, pycparser
+, pyparsing
+, z3-solver
+}:
+buildPythonPackage rec {
+  pname = "miasm";
+  version = "0.1.3-unstable-2024-04-28";
+  format = "setuptools";
```

```suggestion
  pyproject = true";
```

and then `  build-system = [ setuptools ];`.